src/lib.rs

```diff
@@ -61,9 +65,116 @@ impl State {
     }
 }
 
+#[cfg(feature="std")]
+mod oscbuf {
+    use super::MAX_OSC_RAW;
+    pub const DEFAULT_SIZE_LIMIT: usize = u32::max_value() as usize;
```

Comment:
If we're concerned about a DOS attack by applications running inside the terminal, it seems like `u32::max_value()` is a bad value to choose here.